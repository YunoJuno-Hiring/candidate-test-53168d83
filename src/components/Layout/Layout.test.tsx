import React from 'react';
import { Layout } from '@components';
import renderer from 'react-test-renderer';

describe('Layout component', () => {
  it('renders correctly with children', () => {
    const component = renderer
      .create(
        <Layout>
          <div>child div</div>
        </Layout>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
