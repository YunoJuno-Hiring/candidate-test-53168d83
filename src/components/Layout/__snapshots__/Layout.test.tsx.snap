// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component renders correctly with children 1`] = `
<div
  className="Layout-container"
>
  <div>
    child div
  </div>
</div>
`;
